addons/base_import_module/models/ir_module.py

```diff
@@ -194,6 +196,36 @@ def _import_module(self, module, path, force=False, with_demo=False):
             'res_id': asset.id,
         } for asset in created_assets])
 
+        def get_po_paths(module_name, lang, env=None):
```

Comment:
please refactor the `get_po_paths` in the translate.py

Reply:
Why we cannot do this in master?
and the fallback logic for spanish is different from
https://github.com/odoo/odoo/commit/e0ee100b55c58a8d0968cf5c536a4cdb0d72b667

Reply:
> Why we cannot do this in master? 

Because we need a solution for this in 17.0. No industry module is translated... and 17.0 is going to be supported some more years.

> and the fallback logic for spanish is different from [e0ee100](https://github.com/odoo/odoo/commit/e0ee100b55c58a8d0968cf5c536a4cdb0d72b667)

Well spotted! That's the risk with duplicated code. I didn't see the changes in the other PR, as they can evolve independently.
I'll create the PR in master right away so that a cleaner solution is merged there.

Reply:
https://github.com/odoo/odoo/pull/172409 could you have a look?

Reply:
Oh, sorry for my typo
I meant Why we cannot refactor `get_po_paths` in 17.0 by adding an extra parameter env?

Reply:
Well, at first I thought that "stable=no change of a method signature". But you're right that this is not a Class method, so it should be harmless indeed... I'll update this one, thanks.